### DISCUSSION TITLE
Add a config option not to append trailing slash in node adapter (standalone mode)

### DISCUSSION THREAD

**Discussion by mef** [withastro/roadmap#526] [Proposal]
## Summary

The `standalone` mode of `@astrojs/node` currently always appends a forward slash at the end of pages URLs. It would be great to have this as optional, for users who prefer to have their URLS not ending with a slash.

### Current situation

A function `appendForwardSlash`  is called to append slashes to all URLs (inside [standalone.ts](https://github.com/withastro/astro/blob/main/packages/integrations/node/src/standalone.ts#L15)).

### To-be situation

* a `trailingSlash` configuration option is added to the node adapter (to be determined: accepted values: boolean, or `'always' | 'never' | 'ignore'` like in the dev server?)
* `appendForwardSlash` is only called if the option is active

## Read also

#305 outlines a related issue where sitemaps point to a URL causing a 301 redirect, instead of the suitable location.

**Comment by florian-lefebvre** [comment]
I can't find `appendForwardSlash` in the file you linked now so I assume it has been fixed. If it's not the case, open an issue on the main repo!
